src/module/scene/token-document/document.ts

```diff
@@ -312,8 +290,33 @@ class TokenDocumentPF2e<TParent extends ScenePF2e | null = ScenePF2e | null> ext
         }
     }
 
+    /** Synchronize the token image with the actor image if the token does not currently have an image */
+    static setDefaultImage(token: TokenDocumentPF2e | PrototypeTokenPF2e<ActorPF2e>): void {
```

Comment:
assignDefaultImage()? set implies a setter method where the parameter is what you're setting it to.